.aiexclude

```diff
@@ -0,0 +1,3 @@
+fastlane/.env
+local.properties
+constant.dart
```

Comment:
does this look for this file in any subdirectory?

Reply:
I believe this should be

```
**/**/local.properties
**/**/constant.dart
```

Reply:
If I understand the [example](https://developer.android.com/studio/preview/gemini/aiexclude#examples) correctly, it does: 
>The pattern KEYS blocks all files called "KEYS" with no file extension at or below the directory that contains the .aiexclude file.

Reply:
Makes sense! I just saw a big caution in https://developer.android.com/studio/preview/gemini/aiexclude#how-write with the same. 

I was confused by the main screenshot in that link because they do use `*.txt`. Sorry for the noise!

Reply:
No worries, good to double check!